includes/wf_crm_webform_postprocess.inc

```diff
@@ -1705,7 +1705,13 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
       $url = url('node/' . $this->node->nid, array('absolute' => TRUE));
     }
     elseif ($url == '<confirmation>') {
-      $url = url("node/{$this->node->nid}/done", array('query' => array('sid' => $this->submission->sid), 'absolute' => TRUE));
+      global $user;
+      $query = array('sid' => $this->submission->sid);
+      // Add token if user is not authenticated, inline with 'webform_client_form_submit()'
+      if ((int) $user->uid === 0) {
```

Comment:
This line could be simplified as `if (user_is_anonymous())` and then `global $user;` could be removed.